# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1281031312 25200
#      Thu Aug 05 11:01:52 2010 -0700
# Node ID bf000a56a9857a3712e28fafc96682403bb9e1f8
# Parent  2224236440c0164c2fae7cf33f54f1bbb33b4348
configure.ac: Change warning messages about freetype and fontconfig
to reflect their requirement for native graphics.  Bug #30594.

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2010-08-05  Rik <octave@nomad.inbox5.com>
+
+	* configure.ac: Change warning messages about freetype and fontconfig
+	to reflect their requirement for native graphics.  Bug #30594.
+
 2010-08-05  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Remove distribution of emacs directory
 
 	* emacs/info-emacs-info, emacs/info-emacs-octave-help, emacs/module.mk,
 	emacs/NEWS, emacs/octave-hlp.el, emacs/octave-inf.el,
 	emacs/octave-mod.el, emacs/octave-tags, emacs/octave-tags.1,
 	emacs/README, emacs/TODO: Remove from further hg checkouts.
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -810,44 +810,44 @@ else
   MAGICK_LIBS=
 fi
 AC_SUBST(MAGICK_CPPFLAGS)
 AC_SUBST(MAGICK_LDFLAGS)
 AC_SUBST(MAGICK_LIBS)
 
 # ---------------------------------------------------------------------
 
-## libraries needed for graphics
+## libraries needed for native graphics renderer
 
 warn_freetype=""
 
 OCTAVE_OPENGL
 GRAPHICS_LIBS=
 GRAPHICS_CFLAGS=
 
 if test -n "$OPENGL_LIBS"; then
   AC_DEFINE(HAVE_OPENGL, 1, [Define if OpenGL is available])
 
   ## freetype 2
 
   AC_CHECK_FT2([9.0.3], [AC_DEFINE(HAVE_FREETYPE, 1, [Define to 1 if you have Freetype library.])
     XTRA_CXXFLAGS="$XTRA_CXXFLAGS $FT2_CFLAGS"],
-    [warn_freetype="FreeType library not found.  Native renderer will not have on-screen text"])
+    [warn_freetype="FreeType library not found.  Native graphics will be disabled."])
 
   ## fontconfig library
 
   warn_fontconfig=""
   if test -z "$warn_freetype"; then
     PKG_CHECK_MODULES(FONTCONFIG, [fontconfig], [
       have_fontconfig=yes
       OPENGL_LIBS="$FONTCONFIG_LIBS $OPENGL_LIBS"
       XTRA_CXXFLAGS="$XTRA_CXXFLAGS $FONTCONFIG_CFLAGS"
       AC_DEFINE(HAVE_FONTCONFIG, 1, [Define to 1 if fontconfig is present])], [
       have_fontconfig=no
-      warn_fontconfig="Fontconfig not found.  Native text rendering will use hard-coded font instead."])
+      warn_fontconfig="Fontconfig library not found.  Native graphics will be disabled."])
   fi
 
   ## fltk (www.fltk.org)
   AC_ARG_WITH(fltk-prefix,
           [  --with-fltk-prefix=PFX   Prefix where FLTK is installed (optional)],
           fltk_prefix="$withval",
           fltk_prefix="")
   AC_ARG_WITH(fltk-exec-prefix,
